GAC/helpers.py

```diff
@@ -82,3 +82,34 @@ def update(target, source, rate):
     for t, s in zip(target_params, source_params):
         t.assign(t * (1.0 - rate) + s * rate)
 
+
+class FNN(tf.Module):

```

Comment:
You can put this in networks. It also doesn't look necessary since you can just put this stuff in the actual value and critic networks.

Reply:
I tried, but  later I realized that training the 2 networks in critic gonna be trouble so I defined a FNN 
 class....

Reply:
The point is, I wanna split the trainable variables.